.github/workflows/coverage.yml

```
@@ -39,7 +39,7 @@ jobs:
         id: coverage
 
       - name: Report code coverage
-        uses: zgosalvez/github-actions-report-lcov@v3
+        uses: zgosalvez/github-actions-report-lcov@v4.1.23
```

It's better to just use `v4` to avoid updating for each minor versions.
```suggestion
        uses: zgosalvez/github-actions-report-lcov@v4
```